session/locale_sticky_session.rst

```diff
@@ -164,13 +164,6 @@ event::
                 $this->session->set('_locale', $user->getLocale());
             }
         }
-
-        public static function getSubscribedEvents()
```

Comment:
Now that this method is gone, we can also remove the `use Symfony\Component\EventDispatcher\EventSubscriberInterface;` at the top of this class. Thanks!

Reply:
@javiereguiluz Thanks, Yes you're right. It's done. I removed the useless "use".

Reply:
Thanks, it's done.